setup/storage/s3.md

```diff
@@ -5,4 +5,39 @@ sidebar:
   nav: setup
 ---
 
-Documentation for S3 setup with Halyard is coming soon!
+{% include toc %}
+
+Using [S3](https://aws.amazon.com/s3/) as a 
+storage source means that Spinnaker will store all of its persistent data in a
+[Bucket](https://aws.amazon.com/s3/details/). 
+
+## Prerequisites
+
+You need an [AWS Account](https://aws.amazon.com/account/), and an associated
+access key (secret access key & access key id). 
+
+## Editing your Storage Settings
+
+Halyard will create a bucket for you if the bucket you specify doesn't exist
+yet, or if you don't specify one at all. However, if this is the case, you need
```

Comment:
The phrase "if this is the case" follows an "A or B" statement, making it ambiguous to what "this" is referring to. I suggest restating to: "If you specify the bucket, ensure to also supply the bucket's region"